data/macos.yaml

```diff
@@ -11,6 +11,21 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#System_Logs'
 ---
+name: MacOSAppleUnifiedLog
+doc: Apple Unified Log
```

Comment:
Unified Log => Unified Logging / Unified Logging and Activity Tracing / Unified Logging and Tracing System

Reply:
changed to "Apple Unified Logging and Activity Tracing"